controllers/managedocs_controller.go

```diff
@@ -223,3 +227,31 @@ func (r *ManagedOCSReconciler) setStorageClusterParamsFromSecret(sc *ocsv1.Stora
 	sc.Spec.StorageDeviceSets[0].Count = sdsCount
 	return nil
 }
+
+func (r *ManagedOCSReconciler) updateComponents(req ctrl.Request) v1.ComponentStatusMap {
+	var components v1.ComponentStatusMap
+
+	// Checking the StorageCluster component.
+	var sc ocsv1.StorageCluster
+
+	scNamespacedName := types.NamespacedName{
+		Name:      storageClusterName,
+		Namespace: req.Namespace,
```

Comment:
Please use `r.namespace instead`. I intentionally remove the `req` param from `reconcilePhases`